src/main/java/io/druid/guice/JsonConfigurator.java

```diff
@@ -82,7 +82,7 @@ public JsonConfigurator(
           // If it's a String Jackson wants it to be quoted, so check if it's not an object or array and quote.
           String modifiedPropValue = propValue;
           if (! (modifiedPropValue.startsWith("[") || modifiedPropValue.startsWith("{"))) {
-            modifiedPropValue = String.format("\"%s\"", modifiedPropValue);
+            modifiedPropValue = jsonMapper.writeValueAsString(propValue);
```

Comment:
can this change cause any existing config to break, any backwards incompatiblity  ?